Services/AuthenticationService/Validators/UserEmailPasswordValidator.cs

```diff
@@ -13,7 +13,7 @@ public UserEmailPasswordValidator()
                 .EmailAddress()
                 .WithMessage("Email address must be in valid format.");
 
-            RuleFor(user => user.Password)
+            RuleFor(user => user.PasswordHash)
                 .NotEmpty()
                 .WithMessage("Password must not be empty.")
                 .Length(5, 50)
```

Comment:
Will the password hash have the same length criteria as the password?

Reply:
So far we decided to use SHA, so the hash will be of fixed length (40 symbols).
Changed validator in auth service and tests for this validator